homeassistant/components/switcher_kis/strings.json

```
@@ -63,6 +63,14 @@
       "temperature": {
         "name": "Current temperature"
       }
+    },
+    "switch": {
+      "child_lock": {
+        "name": "Lock physical controls"
```

https://github.com/home-assistant/core/pull/132434#discussion_r1872875820

```suggestion
        "name": "Child lock"
```

---

It is done.